.spacemacs

```diff
@@ -332,7 +332,11 @@ you should place your code here."
        web-mode-indent-style 2
        web-mode-markup-indent-offset 2
        web-mode-scss-indent-offset 2
-       css-indent-offset 2)
+       web-mode-markup-indent-offset 2
```

Comment:
This line seems to be duplicated on L333?

Reply:
Fixed :+1: